.github/workflows/BTA_workflow.yml

```diff
@@ -87,9 +87,9 @@ jobs:
   
     - name: BTA workflow test
       run: |
-         python runner.py --wf BTA --json metadata/test_bta_run3.json --campaign Summer22Run3 --executor iterative --isJERC
+         python runner.py --wf BTA --json metadata/test_bta_run3.json --campaign Summer22Run3 --executor iterative 
```

Comment:
don't we want to overwrite any existing files as well like it is done for the non-BTA-like CI pipelines?